src/test/scala/pt/tecnico/dsi/neutron/PortsSpec.scala

```diff
@@ -0,0 +1,28 @@
+package pt.tecnico.dsi.neutron
+
+import cats.effect.IO
+import pt.tecnico.dsi.neutron.models.{Network, Port}
+import pt.tecnico.dsi.neutron.services.BulkCreate
+
+class PortsSpec extends CrudSpec[Port]("port", _.ports) with BulkCreateSpec[Port] {
+
+  val bulkService: NeutronClient[IO] => BulkCreate[IO, Port] = _.ports
+  val updateStub: Port.Update = Port.Update(name = Some("cool-port"))
+  val createStub: IO[Port.Create] = for {
+    neutron <- client
+    net <- neutron.networks.create(Network.Create())
+  } yield Port.Create(name = Some("hello"), networkId = net.id)
+
+  "Ports service" should {
+    "update" in {
+      for {
+        neutron <- client
+        stub <- createStub
+        port <- neutron.ports.create(stub)
+        updated <- neutron.ports.update(port.id, updateStub)
+      } yield assert {
+        updated.name == updateStub.name.get
```

Comment:
Codacy found an issue: [Usage of get on optional type.](https://app.codacy.com/gh/ist-dsi/scala-neutron-client/pullRequest?prid=5827787)

Reply:
OptionValues (see above)